index.js

```diff
@@ -21,6 +21,41 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+    Recipe.insertMany(data)
+    .then((data)=>{
+      for(let i =0 ; i<data.length;i++){
+        console.log(data[i].title)
+      }
+
+    Recipe.findOneAndUpdate({title:"Rigatoni alla Genovese",},{duration:100,},{
+      new:true,
+    })
+    .then(()=>{
+      console.log("UPDATE RECIPE!")
+    })
+    .catch((e)=>{
+      console.log("EStas teniendo un error",e)
+    })
+    
+    Recipe.deleteOne({title:"Carrot Cake"},{new:true})
+    .then(()=>{
+      console.log("ELIMINADO")
+    })
```

Comment:
Tambien podemos ejecutar una promesa despues de otra para que agreguemos el cierre de nuestra db```suggestion
    }).then(() => {
            const closePr = mongoose.connection.close();
            return closePr;
          })
          .then(() => {
            console.log("DATABASE CLOSED!");
          });
```